tests/query/injections/html/test-html-injections.html

```diff
@@ -6,50 +6,50 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link href="css/style.css" rel="stylesheet">
     <style> footer{ } </style>
-    <!--      ^ css  -->
+    <!--       ^ @css  -->
     <style title="Test Style without type attribute"> footer{ } </style>
-    <!--                                                ^ css  -->
+    <!--                                                 ^ @css  -->
     <style type="text/css" title="test style with defined type attribute"> footer{ } </style>
-    <!--                                                                    ^ css  -->
+    <!--                                                                     ^ @css  -->
   </head>
   <body>
     <script> const x = 1 </script>
-           <!-- ^ javascript -->
+           <!--  ^ @javascript -->
```

Comment:
I'd say it's lucky that despite all `@..` is moved 1 space to the right here, it's still inside the intended range.

Reply:
Not lucky, tested ;)